src/test/kotlin/org/axonframework/intellij/ide/plugin/markers/handlers/DeadlineHandlerMethodLineMarkerProviderTest.kt

```diff
@@ -43,7 +43,7 @@ class DeadlineHandlerMethodLineMarkerProviderTest : AbstractAxonFixtureTestCase(
         )
         assertThat(hasLineMarker(DeadlineHandlerMethodLineMarkerProvider::class.java)).isTrue
         assertThat(getLineMarkerOptions(DeadlineHandlerMethodLineMarkerProvider::class.java)).containsExactly(
-            OptionSummary("MyCommand", null, AxonIcons.Publisher)
+            OptionSummary("MyAggregate.handle(MyCommand, DeadlineManager):${actualLineNumber(9)}", "my_special_deadline", AxonIcons.Publisher)
```

Comment:
I guess it's the `toViewText` method you've added, causing the line number to be added, but I don't clearly see how. Would you mind elaborating on this?